### PR TITLE
fix bug: Rotated UIImage lose origin scale and blured.

### PR DESCRIPTION
**Pull request by yanpanpan**
Fix bug. When UIImage rotated, it lost origin scale. 

## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 5.0.
- [x] New extensions support iOS 10.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

**Review by guykogus — CHANGES_REQUESTED** [review]
Please add a CHANGELOG entry and perhaps update the tests

**Comment by yanpanpan** [comment]
CHANGELOG entry is added, all tests is OK and passed.

File: Sources/SwifterSwift/UIKit/UIImageExtensions.swift [hunk 1]
```diff
@@ -169,7 +169,7 @@ public extension UIImage {
                                      width: destRect.width.rounded(),
                                      height: destRect.height.rounded())
 
-        UIGraphicsBeginImageContext(roundedDestRect.size)
+        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, false, scale)
```
Comment (LucianoPAlmeida) [inline Sources/SwifterSwift/UIKit/UIImageExtensions.swift:172]:
```suggestion
        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, /*opaque=*/false, scale)
```

File: Sources/SwifterSwift/UIKit/UIImageExtensions.swift [hunk 2]
```diff
@@ -139,7 +139,7 @@ public extension UIImage {
...
```
Comment (LucianoPAlmeida) [inline Sources/SwifterSwift/UIKit/UIImageExtensions.swift:142]: Just to make it easier for the reader ... I had to go to docs to remember what `false` meant 
```suggestion
        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, /*opaque=*/false, scale)
```

**Review by LucianoPAlmeida — DISMISSED** [review]
LGTM! Thanks :)

**Review by guykogus — APPROVED** [review]
I personally strongly dislike the `/*opaque*/` comments, but 🤷‍♂️

**Comment by LucianoPAlmeida** [comment]
> I personally strongly dislike the `/*opaque*/` comments, but 🤷‍♂️

Ahh feel free to request changes, I personally find useful for literals and use a lot at work with C++ because functions don't have labels there, so we use this pattern to make it easier to read at call site what the parameter means e.g. `p->function(param, /*argument=*/nullptr)` ... 
But feel free to revert ... it's not that important 

@yanpanpan it is good to go can you just revert this so we can merge? :)

File: Sources/SwifterSwift/UIKit/UIImageExtensions.swift [hunk 3]
```diff
@@ -139,7 +139,7 @@ public extension UIImage {
                                      width: destRect.width.rounded(),
                                      height: destRect.height.rounded())
 
-        UIGraphicsBeginImageContext(roundedDestRect.size)
+        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, /*opaque=*/false, scale)
```
Comment (guykogus) [inline Sources/SwifterSwift/UIKit/UIImageExtensions.swift:142]:
```suggestion
        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, false, scale)
```

File: Sources/SwifterSwift/UIKit/UIImageExtensions.swift [hunk 4]
```diff
@@ -169,7 +169,7 @@ public extension UIImage {
...
```
Comment (guykogus) [inline Sources/SwifterSwift/UIKit/UIImageExtensions.swift:172]:
```suggestion
        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, false, scale)
```

**Review by guykogus — CHANGES_REQUESTED** [review]
Ok fine, let's get rid of it...

**Comment by yanpanpan** [comment]
> > I personally strongly dislike the `/*opaque*/` comments, but 🤷‍♂️
> 
> Ahh feel free to request changes, I personally find useful for literals and use a lot at work with C++ because functions don't have labels there, so we use this pattern to make it easier to read at call site what the parameter means e.g. `p->function(param, /*argument=*/nullptr)` ...
> But feel free to revert ... it's not that important
> 
> @yanpanpan it is good to go can you just revert this so we can merge? :)

I personally  think removing `/*opaque*/` looks more `SwifterSwift`. Thanks everyone. Revert is done.

**Review by LucianoPAlmeida — APPROVED** [review]
Thank you!

**Review by guykogus — APPROVED** [review]

**Comment by SD10** [comment]
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃

**Pull request merged**